src/EnergyPlus/WindowManager.hh

```diff
@@ -356,40 +358,6 @@ namespace Window {
                          Array1A<Real64> p  // Blind properties (equivalent to ST_LAY)
     );
 
-    Real64 InterpProfAng(Real64 ProfAng,           // Profile angle (rad)
-                         Array1S<Real64> PropArray // Array of blind properties
-    );
-
-    Real64 InterpSlatAng(Real64 SlatAng,           // Slat angle (rad)
-                         bool VarSlats,            // True if slat angle is variable
-                         Array1S<Real64> PropArray // Array of blind properties as function of slat angle
-    );
-
-    Real64 InterpProfSlatAng(Real64 ProfAng,           // Profile angle (rad)
-                             Real64 SlatAng,           // Slat angle (rad)
-                             bool VarSlats,            // True if variable-angle slats
-                             Array2A<Real64> PropArray // Array of blind properties
-    );
-
-    Real64 BlindBeamBeamTrans(Real64 ProfAng,        // Solar profile angle (rad)
-                              Real64 SlatAng,        // Slat angle (rad)
-                              Real64 SlatWidth,      // Slat width (m)
-                              Real64 SlatSeparation, // Slat separation (distance between surfaces of adjacent slats) (m)
-                              Real64 SlatThickness   // Slat thickness (m)
-    );
-
-    constexpr Real64 InterpProfSlat(Real64 const SlatLower,
-                                    Real64 const SlatUpper,
-                                    Real64 const ProfLower,
-                                    Real64 const ProfUpper,
-                                    Real64 const SlatInterpFac,
-                                    Real64 const ProfInterpFac)
-    {
-        Real64 ValA = SlatLower + SlatInterpFac * (SlatUpper - SlatLower);
-        Real64 ValB = ProfLower + SlatInterpFac * (ProfUpper - ProfLower);
-        return ValA + ProfInterpFac * (ValB - ValA);
-    }
```

Comment:
:+1: :+1: :+1: great to see reuse.  I wonder where else we can cut out different interpolation functions.